components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java

```diff
@@ -2039,6 +2042,9 @@ public void addFileToDocumentation(APIIdentifier apiId, Documentation documentat
                 String[] visibleRoles = new String[0];
                 if (visibleRolesList != null) {
                     visibleRoles = visibleRolesList.split(",");
+                    for(int i = 0; i < visibleRoles.length; i++) {
```

Comment:
Formatting issues

Reply:
Lets also use enhanced for loop(https://stackoverflow.com/a/18738765/3296350), its a lot cleaner